app/models/client.rb

```diff
@@ -198,7 +198,7 @@ class Client < ApplicationRecord
       indexes :analytics_dashboard_url, type: :text
       indexes :analytics_tracking_id, type: :text
       indexes :cumulative_years, type: :integer, index: "false"
-      indexes :subjects, type: :object, properties: {
+      indexes :subjects,  type: :nested, include_in_parent: true,  properties: {
```

Comment:
Likewise here, could this be switched back to being an object type?

Reply:
Will do